backend/src/main/java/edu/netcracker/backend/service/impl/StatisticsService.java

```diff
@@ -0,0 +1,46 @@
+package edu.netcracker.backend.service.impl;
+
+import edu.netcracker.backend.dao.StatisticsDAO;
+import edu.netcracker.backend.message.response.CarrierStatisticsResponse;
+import edu.netcracker.backend.message.response.ServiceDistributionElement;
+import edu.netcracker.backend.message.response.TripDistributionElement;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Service
+public class StatisticsService {
```

Comment:
Please change StatisticsService like interfase and implent it by StatisticsServiceImpl